platform_versioned_docs/version-24.2/data_studios/index.mdx

```diff
@@ -26,6 +26,10 @@ Before you get started, you need the following:
 Currently, Data Studios only supports AWS Batch compute environments that **do not** have Fargate enabled.
 :::
 
+## Limitations
+
+If you configured your AWS Batch compute environment to include an EFS file system with **EFS file system > EFS mount path**, the mount path cannot be the same as your work directory. When the work directory and EFS mount path are identical, the Data Studio snapshots fail causing a data studio instance to fail. For more information on AWS Batch configuration, see [AWS Batch][aws-batch].
```

Comment:
```suggestion
If you configured your AWS Batch compute environment to include an EFS file system with **EFS file system > EFS mount path**, the mount path cannot be the same as your work directory as this will cause the Data Studio snapshots, and session, to fail. The mount path cannot be left empty, as this will default to the work directory and also result in snapshot and session failure.

For more information on AWS Batch configuration, see [AWS Batch][aws-batch]. 
```

Reply:
@endre-seqera, does the suggested wording work?

Reply:
> If you configured your AWS Batch compute environment to include an EFS file system with **EFS file system > EFS mount path**, the mount path cannot be the same as your work directory as this causes the Data Studio snapshots, and session, to fail. You must specify a mount path for the EFS file system or the mount path is set to your work directory.

maybe

Reply:
Last sentence is highly confusing (for me, at least)

Reply:
@endre-seqera, I'm trying to avoid too much of a run-on sentence.

What does it mean for the session to fail exactly? It crashes? There aren't any checkpoints saved and user work vanishes after stopping a session?

Reply:
In this case failure means that the data studios session will fail to start. As part of the startup process the Compute Environment's work directory is used to set up snapshot creation. However, since EFS is not supported for this, the startup process fails.